pkg/utils/openshift/utils.go

```diff
@@ -48,6 +51,69 @@ func MapKnownVersion(info platform.PlatformInfo) platform.OpenShiftVersion {
 		"1.10+": "3.10",
 		"1.11+": "3.11",
 		"1.13+": "4.1",
+		"1.14+": "4.2",
+		"1.16+": "4.3",
```

Comment:
let's drop this, it'll be added by Abhishek's PR and we won't need after changes anyways

Reply:
Is this our current best choices : maintain a mapping?  I think so.